Accept a wider range of versions, the provided ones are too
restrictive and sometimes outside the gentoo tree.
Patch by Nicola Smaniotto.

diff --git a/requirements.txt b/requirements.txt
index 586b007..987eeb2 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -1,5 +1,5 @@
-sentry_sdk==1.5.3
-yt-dlp>=2022.01.21
-spotipy==2.16.1
-mutagen==1.45.1
-rich==11.0.0
\ No newline at end of file
+sentry_sdk>=1.5, <2
+yt-dlp>=2021.10.27
+spotipy>=2.16, <3
+mutagen>=1.45, <2
+rich>=11.0, <12
